.buildkite/pipeline.yml

```diff
@@ -4,7 +4,7 @@ env:
   ELASTIC_PACKAGE_COMPOSE_DISABLE_ANSI: "true"
   ELASTIC_PACKAGE_COMPOSE_DISABLE_PULL_PROGRESS_INFORMATION: "true"
   KIND_VERSION: 'v0.20.0'
```

Comment:
Also kind is rather old. v1.27.3 is latest

Reply:
[Here](https://github.com/kubernetes-sigs/kind/releases) it says the latest version for kind is 0.20.0, maybe you are mixing up with the kindest version for the cluster?